map_view/display-layer-view-state/src/main/java/com/esri/samples/display_layer_view_state/DisplayLayerViewStateSample.java

```diff
@@ -101,60 +93,50 @@ public void start(Stage stage) {
         EnumSet<LayerViewStatus> layerViewStatus = statusChangeEvent.getLayerViewStatus();
         displayViewStateText(layerViewStatus);
 
-        // if there is an error or warning, format and display the message as an alert
+        // if there is an error or warning, format and display a message as an Alert
         ArcGISRuntimeException error = statusChangeEvent.getError();
         if (error != null) {
           Throwable cause = error.getCause();
-          String message = (cause != null) ? cause.toString() : error.toString();
+          String message = (cause != null) ? "Network issue, unable to load layer" : error.getMessage();
           Alert alert = new Alert(Alert.AlertType.ERROR, message);
           alert.show();
         }
       });
 
       // create buttons to toggle the visibility of the feature layer
-      loadLayerButton = new Button("Load Layer");
-      loadLayerButton.getStyleClass().add("panel-button");
-      hideLayerButton = new Button("Hide Layer");
-      hideLayerButton.getStyleClass().add("panel-button");
-      // initially add load layer button to the control panel
-      controlsVBox.getChildren().add(loadLayerButton);
+      toggleLayerButton = new Button("Load Layer");
 
       // create a listener for clicks on the load layer button
-      loadLayerButton.setOnAction(event -> {
+      toggleLayerButton.setOnAction(event -> {
 
-        // if the feature layer already exists and is hidden, toggle it's visibility to visible
-        if (featureLayer != null && !featureLayer.isVisible()) {
-          featureLayer.setVisible(true);
+        // if the feature layer already exists toggle it's visibility to visible
```

Comment:
Strictly speaking we're not just toggling this to visible, but toggling the visibility on/off. Also suggesting to add a comma as we have one in the `else` block:


```suggestion
        // if the feature layer already exists, toggle it's visibility
```